zenodo/modules/stats/registrations.py

```diff
@@ -148,6 +148,7 @@ def register_queries():
                     recid='recid',
                 ),
                 metric_fields=dict(
+                    count=('sum', 'count'),
```

Comment:
I removed the default `total` (which was just a sum metric for the `count` field) and made it part of the `metric_fields` parameter. So now we have to basically explicitly add it.